terraform/modules/hub/vpc.tf

```diff
@@ -33,7 +33,8 @@ resource "aws_vpc_endpoint" "s3" {
             ],
             "Resource": [
                 "arn:aws:s3:::govukverify-eidas-metadata-aggregator-${var.deployment}-a/*",
-                "arn:aws:s3:::govukverify-eidas-metadata-aggregator-${var.deployment}-a"
+                "arn:aws:s3:::govukverify-eidas-metadata-aggregator-${var.deployment}-a",
```

Comment:
Why `-a` and not `e`? Or are we getting things from/to to the Old World?

Reply:
this is an existing bucket
we're going to change how this works but for now we just proxy to the existing bucket

https://govukverify-eidas-metadata-aggregator-joint-a.s3-eu-west-2.amazonaws.com/